cmake/FindPackages.cmake

```diff
@@ -8,7 +8,7 @@ find_package(Boost REQUIRED REQUIRED program_options)
 find_package(Eigen3 3.3.4 REQUIRED)
 
 # Find libint
-find_package(libint2 REQUIRED)
+find_package(Libint2 REQUIRED)
```

Comment:
I agree with the capitalization: could you do the same for the CMake variables inside ˋFindLibint2.cmakeˋ?